src/style.css

```diff
@@ -1 +1,96 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir.ttc) format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir Book';
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: 'Avenir Heavy';
+  src: url(/src/fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 800;
+}
+
+body {
+  margin: 0 10px;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
```

Comment:
The `font-family` should match the custom font name defined in the `@font-face` rule without including fallback fonts. Please update it to use only the custom font name.